installer/install.iss

```diff
@@ -375,6 +375,11 @@ var
     // Previous Git for Windows version (if upgrading)
     PreviousGitForWindowsVersion:String;
 
+    // List of pages that _only_ have options whose value has been already set (by command line or previous install)
+    PagesPreSet:String;
+    // Tracks whether all options on a page have been pre-set
+    OptionsArePreSet:Boolean;
```

Comment:
If you revert the logic, i.e. if you track the pages that have new options that have not previously been seen by the user (the variable name `PagesWithPreviouslyUnseenOptions` comes to mind), then you need only one variable, and you also need less code changes.